src/ol_dbt/models/marts/combined/_marts__combined__models.yml

```
@@ -82,6 +82,9 @@ models:
 - name: marts__combined__orders
   description: B2B and regular orders combined into one table
   columns:
+  - name: combined_orders_id
+    description: int, primary key for this table. At the grain of order, platform,
+      and line.
```

```suggestion
      tests:
      - not_null
```
Also, can we update the description to match the order of the long cast in the SQL file and give an example of the generated ID?